content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,152 @@
+| `sudo service datadog-agent flare`              | `sudo datadog-agent flare`              | Send flare |
+| `sudo service datadog-agent`                    | `sudo datadog-agent --help`             | Display command usage |
+| `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent -- datadog-agent check <check_name>` | Run a check |
+{{% /table %}}
 
-To stop the Agent:
-```shell
-sudo /etc/init.d/datadog-agent stop
-```
+**NB**: If `service` is not available on your system, use:
 
-To restart the Agent and reload the configuration files:
-```shell
-sudo /etc/init.d/datadog-agent restart
-```
+* on `upstart`-based systems: `sudo start/stop/restart datadog-agent`
+* on `systemd`-based systems: `sudo systemctl start/stop/restart datadog-agent`
+* on `initctl`-based systems: `sudo initctl start/stop/restart datadog-agent`
 
-## Status and Information
-To check if the Agent is running: *(since 3.8.0)*
-```shell
-sudo /etc/init.d/datadog-agent status
+## Configuration
+
+The configuration files and folders for for the Agent is located at:
+
+| Agent v5                                  |  Agent v6                          |
+|:-----|:----|
+|`/etc/dd-agent/datadog.conf`| `/etc/datadog-agent/datadog.yaml` |
+
+Configuration files for [integrations](/integrations):
+
+| Agent v5                                  |  Agent v6                          |
+|:-----|:----|
+|`/etc/dd-agent/conf.d/`|`/etc/datadog-agent/conf.d/`|
+
+## Troubleshooting
+
+Run the info or status command to see the state of the Agent.
+The Agent logs are located in the `/var/log/datadog/` directory:
+
+* For Agent v6 all logs are in the `agent.log` file
+* For Agent v5 logs are in:
+    
+    * `datadog-supervisord.log`
+    * `collector.log`
+    * `dogstatsd.log`
+    * `forwarder.log`
+
+If you're still having trouble, [our support team](/help) will be glad to provide further assistance.
+
+## Adding a custom python package to the agent
+
+The Python interpreter embedded with the Agent is located here:  
+`/opt/datadog-agent/embedded/bin/python`.
+The agent also comes with pip, install python libraries using:
 ```
-To receive information about the Agent’s state:
-```shell
-sudo /etc/init.d/datadog-agent info
+sudo -u dd-agent -- /opt/datadog-agent/embedded/bin/pip install <package_name>
 ```
 
-Tracebacks for errors can be retrieved by setting the `-v` flag: (since 3.8.0)
-```shell
-sudo /etc/init.d/datadog-agent info -v
-```
+## Switch between Agent v5 and v6
+### Upgrade to Agent 6
+
+A script is available to automatically install or upgrade the new Agent. It sets up the repos and install the package for you; in case of upgrade, the import tool also searches for an existing `datadog.conf` from a prior version and converts Agent and checks configurations according to the new file format and filesystem layout.
+#### One-step install
+##### To Upgrade
+
+In case you have an Agent version 5.17 or later and you want to import the
```

Comment:
Whats the tie to 5.17?  

The Agent 6.x installer can automatically convert your 5.x style agent configuration at upgrade:

Reply:
👌